### PR TITLE
chore(github): added dependabot

### PR DESCRIPTION
**Pull request by carlocorradini**
Related to PR #671 

What has been changed:

- Added [`Dependabot`](https://github.com/dependabot)

### CODE REVIEW

**Comment by carlocorradini** [comment]
@henil @de-sh  Any update? 🤗🥳

**Comment by carlocorradini** [comment]
@swanandx @henil Any thoughts on this? I apologize for pinging

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      schedule:
+          interval: "weekly"
+      ignore:
+          - dependency-name: "*"
+            update-types: ["version-update:semver-patch"]
+
+    - package-ecosystem: "github-actions"
+      directory: "/"
+      schedule:
+          interval: "weekly"
```
Comment (henil) [inline .github/dependabot.yml:24]: please remove this, we only need cargo one

Comment (carlocorradini) [reply .github/dependabot.yml:24]: Could you please explain why you don't want GitHub action updates? Thanks 😅

Comment (henil) [reply .github/dependabot.yml:24]: ideally we should update github-actions, but updating it sometime can be alot of work. so i prefer to keep what works until it keeps working :)

Comment (carlocorradini) [reply .github/dependabot.yml:24]: Well... GitHub action files are few tenth/hundred of lines.
It should not be a big deal 😂🤯

**Pull request closed without merging**